extensions/README.md

```diff
@@ -260,20 +260,24 @@ See [Publicize](blocks/publicize/index.js) and [Shortlinks](blocks/shortlinks/in
 
 ### i18n
 
-As of 04/2019, `wp.i18n` [doesn't support React elements in strings](https://github.com/WordPress/gutenberg/issues/9846). You will have to structure your copy so that links and other HTML can be translated separately.
-
-Not possible:
-
-```js
-__( 'Still confused? Check out <a>documentation</a> for more!' )
-```
-
-Possible:
+`@wordpress/i18n` doesn't support React elements in strings, but you can use `__experimentalCreateInterpolateElement` from `@wordpress/element`:
 
 ```jsx
-{ __( 'Still confused?' ) } <a>{ __( 'Check out documentation for more!' ) }</a>
+import { BlockIcon } from '@wordpress/block-editor';
+import { __experimentalCreateInterpolateElement as createInterpolateElement } from '@wordpress/element';
+const getDocumentationLink = () => {
+	return createInterpolateElement(
+		__( '<FlagIcon /> Still confused? <a>Check out documentation for more!</a>', 'jetpack' ),
+		{
+			FlagIcon: <BlockIcon icon={ 'flag' } />,
+			a: <a href={ 'https://jetpack.com' } />,
```

Comment:
`href="https://jetpack.com"` and `icon="flag"` would work. ;-)

Reply:
True. Done in c26b1c98862d4eaf8335a686a958a3591d3595d5